resolwe/flow/filters.py

```diff
@@ -250,6 +266,8 @@ class BaseCollectionFilter(TextFilterMixin, UserFilterMixin, BaseResolweFilter):
     contributor_name = filters.CharFilter(method="filter_contributor_name")
     owners = filters.CharFilter(method="filter_owners")
     owners_name = filters.CharFilter(method="filter_owners_name")
+    shared_with_me = filters.BooleanFilter(method="filter_private")
+    group = filters.CharFilter(method="filter_for_group")
```

Comment:
How exactly is the group filter supposed to bo used?

Reply:
User supplies the ID of the group to filter by. I meant to use `NumberFilter`, but on second thought, `ModelChoiceFilter` seems even more appropriate.